src/pages/LandingPage.tsx

```diff
@@ -78,22 +90,51 @@ export const LandingPage: FunctionComponent = () => {
 
             <PageSection variant={PageSectionVariants.light} className="pf-u-p-md">
               <div className="pf-u-text-align-right">
-                <Button variant="link" icon={<ThIcon />} className="pf-u-mr-sm" isInline isLarge/>
-                <Button variant="link" icon={<ThListIcon />} isInline isLarge isDisabled/>
+                <Button variant="link" icon={<ThIcon />} onClick={() => setView('grid')} className="pf-u-mr-sm" isInline isLarge/>
+                <Button variant="link" icon={<ThListIcon />} onClick={() => setView('list')} isInline isLarge/>
```

Comment:
We should disable the non active ones here or change the color of the non selected ones. Right now, both shows up as blue, which might confuse users.
I think we can use the `useState()` hook to manage the state of that interaction.

Reply:
Agree with @tahmidefaz this would disable the active ones 

```suggestion
                <Button isDisabled={view === 'grid'} variant="link" icon={<ThIcon />} onClick={() => setView('grid')} className="pf-u-mr-sm" isInline isLarge/>
                <Button isDisabled={view === 'list'} variant="link" icon={<ThListIcon />} onClick={() => setView('list')} isInline isLarge/>
```